actions/transfer-case.vbs

```diff
@@ -51,84 +51,124 @@ changelog = array()
 DO
-	BeginDialog Dialog1, 0, 0, 231, 145, "Case Transfer"
-    EditBox 55, 5, 50, 15, MAXIS_case_number
-    EditBox 175, 5, 50, 15, servicing_worker
-    CheckBox 40, 30, 55, 10, "out of county", out_of_county_checkbox
-    CheckBox 95, 30, 55, 10, "within county", in_county_checkbox
-    EditBox 105, 45, 120, 15, transfer_reason
-    EditBox 105, 65, 120, 15, action_taken
-    EditBox 105, 85, 120, 15, requested_verifs
-    EditBox 105, 105, 120, 15, other_notes
-    'CheckBox 5, 130, 75, 10, "Send SPEC/MEMO", SPEC_MEMO_checkbox
-    ButtonGroup ButtonPressed
-    OkButton 120, 125, 50, 15
-    CancelButton 175, 125, 50, 15
-    Text 115, 10, 55, 10, "Transferring to:"
-    Text 5, 30, 35, 10, "Transfer:"
-    Text 5, 50, 70, 10, "Reason for transfer:"
-    Text 5, 70, 70, 10, "Actions taken:"
-    Text 5, 90, 95, 10, "Actions/verifications needed:"
-    Text 5, 110, 45, 10, "Other notes:"
-    Text 5, 10, 50, 10, "Case number:"
-    Text 175, 20, 50, 10, " Ex. (X127ICT)"
-    EndDialog
+    active_worker_found = TRUE
+    Do
+    	Do
+    		err_msg = ""
+    		Dialog Dialog1
+    		cancel_without_confirmation
+            Call validate_MAXIS_case_number(err_msg, "*")
+            IF len(transfer_to_worker) <> 7 THEN err_msg = err_msg & vbNewLine & "* Please enter the new servicing worker."
+            IF trim(worker_signature) = "" THEN err_msg = err_msg & vbNewLine & "* Please enter your worker signature."
+            If ButtonPressed = search_button Then               'Pulling up the hsr page if the button was pressed.
+                run "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe https://hennepin.sharepoint.com/teams/hs-es-manual/sitepages/transfers.aspx?web=1"
+                err_msg = "LOOP"
+            Else                                                'If the instructions button was NOT pressed, we want to display the error message if it exists.
+    		    IF err_msg <> "" THEN MsgBox "*** NOTICE!***" & vbNewLine & err_msg & vbNewLine
+            End If
+    	Loop until err_msg = ""
+        CALL check_for_password(are_we_passworded_out)			'function that checks to ensure that the user has not passworded out of MAXIS, allows user to password back into MAXIS
+    LOOP UNTIL are_we_passworded_out = false					'loops until user passwords back in
+
+    transfer_to_worker = trim(transfer_to_worker)                'formatting the information entered in the dialog
+    transfer_to_worker = Ucase(transfer_to_worker)               'making sure we are capital in all things'
+    worker_number = trim(worker_number)
+    worker_number = Ucase(worker_number)
+
+    '----------Checks that the worker or agency is valid---------- 'must find user information before transferring to account for privileged cases.
+
+    CALL navigate_to_MAXIS_screen_review_PRIV("CASE", "CURR", is_this_priv) ' need discovery on priv cases for xfer handling'
```

Comment:
Recommend moving this (line 134-137) outside of this larger loop so that the script doesn't bounce around in MAXIS too much if we have to loop.

Reply:
moved to line 188 after we are done in REPT/USER